Jellyfin.Api/Auth/BaseAuthorizationHandler.cs

```diff
@@ -106,6 +110,20 @@ public abstract class BaseAuthorizationHandler<T> : AuthorizationHandler<T>
                 return false;
             }
 
+            // User attempting to access LiveTV without permission.
+            if (requireLiveTvAccessPermission
+                && !user.HasPermission(PermissionKind.EnableLiveTvAccess))
```

Comment:
BaseAuthorizationHandler is becoming way too bloated imo and LiveTv is moving way beyond "Base". I would rather we either retrieve the user in `LiveTvManagementHandler` and `LiveTvAccessHandler` (it's cached by the UserManager) or store the User as a nullable protected property in `BaseAuthorizationHandler`, and then handle the LiveTv check in the respective live tv handlers.